docs/api/browser-window.md

```diff
@@ -348,6 +348,9 @@ It creates a new `BrowserWindow` with native properties as set by the `options`.
       You can access this context in the dev tools by selecting the
       'Electron Isolated Context' entry in the combo box at the top of the
       Console tab.
+    * `worldSafeExecuteJavaScript` Boolean (optional) - If true, values returned from `webFrame.executeJavaScript` will be sanitized to ensure JS values
+      can't unsafely cross between worlds when using `contextIsolation`.  The default
+      is `false`. In Electron 12, the default will be changed to `true`. _Deprecated_
```

Comment:
I wonder if this might be a good test case for the `base::Feature`-based stuff we've been talking about in api-wg?

Reply:
Maybe, but I don't super want to figure out that system / pioneer it in this PR

Reply:
Someone's gotta be the first :\